spec/features/idv/steps/verify_step_spec.rb

```diff
@@ -7,10 +7,11 @@
     start_idv_from_sp
```

Comment:
We'll probably want to rename this spec

Reply:
Wait, I was misunderstanding. This adds the jurisdiction step, but doesn't pull it into the flow, correct? Looks like the verify step is still there and takes us to the profile step?

Reply:
Its just a naming issue, the old page is no longer in the flow, I can rename these to be 'jurisdiction_step'